doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -60,6 +79,547 @@ Recommended changes
+
+        - Remove:
+
+          .. code-block:: C++
+
+            /* Initialize LEDs */
+            LEDWidget::InitGpio();
+            LEDWidget::SetStateUpdateCallback(LEDStateUpdateHandler);
+
+            sStatusLED.Init(SYSTEM_STATE_LED);
+
+            UpdateStatusLED();
+
+            /* Initialize buttons */
+            int ret = dk_buttons_init(ButtonEventHandler);
+            if (ret) {
+              LOG_ERR("dk_buttons_init() failed");
+              return chip::System::MapErrorZephyr(ret);
+            }
+
+            /* Initialize function timer */
+            k_timer_init(&sFunctionTimer, &AppTask::FunctionTimerTimeoutCallback, nullptr);
+            k_timer_user_data_set(&sFunctionTimer, this);
+
+        - Add:
+
+          .. code-block:: C++
+
+            if (!Nrf::GetBoard().Init()) {
+                LOG_ERR("User interface initialization failed.");
+                return CHIP_ERROR_INCORRECT_STATE;
+            }
+
+      * Replace the code in the :c:func:`StartApp` method, in the :file:`app_task.cpp` file:
+
+        - Remove:
+
+          .. code-block:: C++
+
+            AppEvent event = {};
+
+            k_msgq_get(&sAppEventQueue, &event, K_FOREVER);
+            DispatchEvent(event);
+
+        - Add in the while loop:
+
+          .. code-block:: C++
+
+            Nrf::DispatchNextTask();
+
+      * Replace the code in the :c:func:`ChipEventHandler` method, in the :file:`app_task.cpp` file:
+
+        - Add at the top of the method:
+
+          .. code-block:: C++
+
+            bool sIsNetworkProvisioned = false;
+            bool sIsNetworkEnabled = false;
+
+        - Remove for the :c:enum:`kCHIPoBLEAdvertisingChange` enum:
+
+          .. code-block:: C++
+
+            sHaveBLEConnections = ConnectivityMgr().NumBLEConnections() != 0;
+            UpdateStatusLED();
+
+        - Add for the :c:enum:`kCHIPoBLEAdvertisingChange` enum:
+
+          .. code-block:: C++
+
+            if (ConnectivityMgr().NumBLEConnections() != 0) {
+              Nrf::GetBoard().UpdateDeviceState(Nrf::DeviceState::DeviceConnectedBLE);
+            }
+
+        - Remove for the :c:enum:`kThreadStateChange` and the :c:enum:`kWiFiConnectivityChange` enums:
+
+          .. code-block:: C++
+
+            UpdateStatusLED();
+
+        - Add for the :c:enum:`kThreadStateChange` and the :c:enum:`kWiFiConnectivityChange` enums:
+
+          .. code-block:: C++
+
+            if (sIsNetworkProvisioned && sIsNetworkEnabled) {
+              Nrf::GetBoard().UpdateDeviceState(Nrf::DeviceState::DeviceProvisioned);
+            } else {
+              Nrf::GetBoard().UpdateDeviceState(Nrf::DeviceState::DeviceDisconnected);
+            }
+
+      * Add the ``board`` and ``task_executor`` modules to the compilation.
+        Edit :file:`CMakeLists.txt` file as follows:
```

Comment:
```suggestion
      #. Add the ``board`` and ``task_executor`` modules to the compilation.
         Edit the :file:`CMakeLists.txt` file as follows:
```
I don't think any competent developer would make the mistake, but do we want to remove the possibility of this being read as removing all previous entries in `target_sources`?

Reply:
I'm not sure if this could be misread, as we write it's about adding new and not replacing the existing sources. What's your opinion?

Reply:
It's the "edit" part that I'm worried about. But I guess since we specifically mention removing code in other sections, it shouldn't be a problem.